README.md

```diff
@@ -20,9 +20,47 @@ make undeploy
 ```
 
 # Use
-Once the operator has been deployed succesfully to the openshift/kubernetes cluster, you can see it in action by creating a new "ansibleee" CR. An example can be found at examples/ansibleee-jlarriba.yaml.
+Once the operator has been deployed succesfully to the openshift/kubernetes cluster, you can see it in action by creating a new "ansibleee" CR. An example can be found at examples/ansibleee-test.yaml.
 ```
-kubectl apply -f examples/ansibleee-jlarriba.yaml
+kubectl apply -f examples/ansibleee-test.yaml
```

Comment:
This name change assumes that the following PR has merged:

  https://github.com/openstack-k8s-operators/ansibleee-operator/pull/4